packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -172,6 +173,21 @@ class VMService {
       });
     }
 
+    _peer.registerMethod('getFlutterVersion', (rpc.Parameters params) async {
+      final FlutterVersion version = FlutterVersion();
+      final Map<String, Object> versionJson = version.toJson();
+      versionJson['frameworkRevisionShort'] = version.frameworkRevisionShort;
+      versionJson['engineRevisionShort'] = version.engineRevisionShort;
+      return versionJson;
+    });
+
+    // If the Flutter Engine doesn't support service registration this will
```

Comment:
Remove this comment and likely remove the other similar obsolete comment in flutter_tools.
Fyi @bkonyi the version of flutter_engine supported by tip of trunk flutter_tools should always support this, right?

Reply:
AFAIK, service registration is always supported

Reply:
done.